File: API_SC_01_08Abril/alanda amabily/api_alanda/api_alanda_amabily/q2_escada_transport.js
import { get_number, print, get_text} from '../utils/io_utils.js'

function main(){
    const peso_atual = get_number('Seu peso atual: ')
    const sexo = get_text('Sexo: ')
    const quilos_perder = get_number('Quantos quilos quer perder: ')
    const dias_semana = get_number('Quantos dias por semana irá fazer atividade física: ')
    const tempo = get_number('Quanto tempo por dia irá treinar: ')
    const tempo_transport = get_number('Proporção de tempo alocada para o transport: ')


    while(quilos_perder !== 0){
        const calorias_perdidas_transport  = calcular_Transport()
    }

    const resultado = `
    -------------------------------------------------
    Alcançará o objetivo desejado em: ${semanas} semanas
    Minutos de escada por dia: ${minutos_escada} minutos
    Minutos de transport por dia: ${minutos_transport}minutos
    --------------------------------------------------
    `
    print(resultado)




}
function calcular_Transport(){
    if (sexo === 'M'){
        return (tempo * 1000) / 5
    }else if(sexo === 'F'){
        return (tempo * 1000) / 6
    }

}
function calcular_escada(){
    if(sexo === 'M'){
        return (tempo *100) / 7
    }else if(sexo === 'F')
    return (tempo * 100)/8
}
main()

File: API_SC_01_08Abril/Thalysson Delano/apisc.py/q2_escada_transport.py
def main():
    
    print('Programa de Perda de Peso\n-------------')

    sexo = str(input('Qual seu Sexo ( M ou F) : ')).upper()
    peso_atual = float(input('Qual seu Peso Atual : '))
    perda_quilos = float(input('Quantos Quilos Deseja Perder : '))
    dias_semana = int(input('Quantos Dias de Treino por Semana : '))
    minutos_dia = int(input('Quanto Tempo de Treino por Dia ( Minutos ) : '))
    proporçao_transport = int(input('Qual a Proporçao de Tempo para Transporte ( Porcentagem ) : '))
    proporçao_escada = 100 - proporçao_transport
    total_calorias_perder = perda_quilos * 7000
    
    calorias_por_semana = 0
    semanas_totais = 0
    while calorias_por_semana < total_calorias_perder:
        semanas_totais += 1
        if sexo == 'M':
            tempo_transport = (minutos_dia / 100) * proporçao_transport
            tempo_escada = (minutos_dia / 100) * proporçao_escada
            calorias_por_dia = (20 * tempo_transport) + (14.3 * tempo_escada)
            calorias_por_semana += calorias_por_dia * dias_semana
        elif sexo == 'F':
            tempo_transport = (minutos_dia / 100) * proporçao_transport
            tempo_escada = (minutos_dia / 100) * proporçao_escada
            calorias_por_dia = (16.67 * tempo_transport) + (12.5 * tempo_escada)
            calorias_por_semana += calorias_por_dia * dias_semana
 
    print(f'''====== RELATORIO ======
    Minutos de Transport por Dia : {tempo_transport} Minutos
    Minutos de Escada por Dia : {tempo_escada} Minutos
    Semanas Necessárias para Atingir a Meta : {semanas_totais} Semanas''')

main()